NextcloudTalk/NCChatViewController.m

```diff
@@ -1328,7 +1356,7 @@ - (void)didPressResend:(NCChatMessage *)message {
     [self removeUnreadMessagesSeparator];
     
     [self removePermanentlyTemporaryMessage:message];
-    [self sendChatMessage:message.message withParentMessage:message.parent];
+    [self sendChatMessage:message.message withParentMessage:message.parent silently:NO];
```

Comment:
Is this obvious to the user? We could just store in NCChatMessage if the message was intended to be silently send and then adjust the resend label and action accordingly?

Reply:
Yeah sounds like a valid idea.
I would also expect this. I tried to send silently, just because there was a network interruption or something the resend shouldn't sent with notifications

Reply:
Hm. My store it on the server side? If we have a temporary message which can be resend, it never hit the server, does it? It would only make sense for message we try to resend, because the call to the server failed. Or am I missing something here?

Reply:
For this particular case you are right, it doesn't make sense to store it server side since the message won't reach the server.
But the information about a message being sent silently or not is missing on the server at the moment.
Once we decide where this information will be stored in the room object, we will be able to modify this attribute in the temporary messages.
I just don't won't to add an attribute (e.g "silent") to the NSRoom object (that means modifying the DB schema) until we decide how it will be stored on the server side :)

Reply:
Follow up discussion in https://github.com/nextcloud/spreed/issues/8234